soroban-sdk/src/storage.rs

```diff
@@ -201,6 +201,57 @@ impl Storage {
             .unwrap_infallible();
     }
 
+    /// Update a value stored against a key.
+    ///
+    /// Loads the value, calls the function with it, then sets the value to the
+    /// returned value of the function.  If no value is stored with the key then
```

Comment:
nit: Seems like you have double-spaces in a few places here.

Reply:
Oops. Annoyingly for some reason when VSCode rewraps the lines it adds two spaces between sentences.